src/protocols/ethereum/views/TransactionDetails.vue

```diff
@@ -123,10 +144,26 @@ export default defineComponent({
     const fee = computed((): number => transaction.value?.tx?.fee || 0);
     const amount = computed((): number => transaction.value?.tx?.amount || 0);
 
+    const transactionArguments = computed(() => decodedCallData?.value?.args?.length
+      ? JsonBig.stringify(decodedCallData.value.args)
+      : undefined);
+
+    async function getDecodedCallData() {
+      if (innerTx.value?.callData && innerTx.value?.recipientId) {
+        return decodeTxData(
+          innerTx.value.callData,
+          innerTx.value.recipientId,
```

Comment:
```suggestion
          innerTx.value.contractId,
```

Should we use `contractId` here?